migrations/20200707214505_add_join_requests_status.js

```diff
@@ -0,0 +1,21 @@
+
+exports.up = async function(knex, Promise) {
+  await knex.schema.table('join_requests', table => {
+    table.integer('status');
+    table.index(['community_id', 'status'])
+  })
+
+  const now = new Date().toISOString()
+  await knex.raw(`UPDATE "join_requests" set status = 2, updated_at = '${now}'`)
+};
+  
+exports.down = async function(knex, Promise) {
+  await knex.schema.table('join_requests', table => {
+    table.dropIndex(['community_id', 'status'])
```

Comment:
dropping the column will automatically drop the index I believe but this is also fine